release/ray_release/environments/prod.env

```diff
@@ -1,5 +1,5 @@
-ANYSCALE_HOST=https://console.anyscale.com
-RELEASE_AWS_ANYSCALE_SECRET_ARN="arn:aws:secretsmanager:us-west-2:029272617770:secret:release-automation/anyscale-token20210505220406333800000001-BcUuKB"
-RELEASE_DEFAULT_CLOUD_ID="cld_4F7k8814aZzGG8TNUGPKnc"
-RELEASE_DEFAULT_PROJECT="prj_FKRmeV5pA6X72aVscFALNC32"
-ANYSCALE_PROJECT="prj_FKRmeV5pA6X72aVscFALNC32"
\ No newline at end of file
+ANYSCALE_HOST=https://console.anyscale-staging.com
```

Comment:
I am not sure we should be turning the prod env into copy of staging

Reply:
True, it's a hack, I will make a new PR to do this correctly